src/main/docs/guide/breaks.adoc

```diff
@@ -0,0 +1,15 @@
+This section documents breaking changes between Micronaut versions
+
+== 4.0.0
+
+===== Repositories validation
```

Comment:
can we also add this as a documentation section in the docs so it is not just kept in breaks